<launch> 
    <param name="use_sim_time" value="true"/>    
    <node pkg="rosbag" type="play" name="player" args="--clock /home/wsk/ROS_Programing/byc_bags/2023-04-24-18-59-24.bag"/>

    <node pkg="cloud_preprocess" type="savecloud" name="savecloud" output="screen">
        <param name="save_enable" value="false"/>
        <param name="input_topic" value="/rslidar_points"/>
        <param name="savefile_path" value="$(env HOME)/mypcd"/>
    </node>
    <!-- <node pkg="rviz" type="rviz" name="rviz" args="-d $(find cloud_preprocess)/pcd.rviz"/>    -->

    <!-- <node pkg="rqt_console" type="rqt_console" name="rqt_console"/> -->

    <!-- <node pkg="rqt_gui" type="rqt_gui" name="rqt_gui"/> -->
</launch> 

